includes/specials/SpecialDataDump.php

```diff
@@ -118,6 +112,16 @@ private function doDelete( string $type, string $fileName ) {
 	}
 
 	private function onDeleteDump( $dbw, $fileName ) {
+
+		if ( $dbw->selectRow(  'data_dump', '*', [ 'dumps_filename' => $fileName ] ) ) {
```

Comment:
Wouldn’t it make more sense to log if the delete completes; rather than log if there is a deletion possible? Would reduce DB queries by one as well